tests/functional/firefox/new/test_sticky_promo.py

```diff
@@ -10,6 +10,7 @@
 @pytest.mark.nondestructive
 def test_sticky_promo(base_url, selenium):
     page = StickyPromoPage(selenium, base_url).open()
+    page.init_promo()
     assert page.promo.is_displayed
     page.promo.close()
     assert not page.promo.is_displayed
```

Comment:
This matches your original plan: 
- Confirm it isn't there
- Scroll down the page
- See it, comfirm
- Dismiss it, confirm 

🚀